### PR TITLE
Ms graph list members bug fix

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/44151)

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.py [hunk 1]
```diff
@@ -224,9 +224,9 @@ def list_groups_command(client: MsGraphClient, args: Dict) -> Tuple[str, Dict, D
         next_link_response = groups['@odata.nextLink']
 
     if next_link_response:
-        entry_context = {f'{INTEGRATION_CONTEXT_NAME}(val.ID === obj.ID).NextLink': next_link_response,
+        entry_context = {f'{INTEGRATION_CONTEXT_NAME}NextLink:': {'GroupsNextLink': next_link_response},
                          f'{INTEGRATION_CONTEXT_NAME}(val.ID === obj.ID)': groups_outputs}
-        title = 'Groups (Note that there are more results. Please use the next_link argument to see them.):'
+        title = 'Groups (Note that there are more results. Please use the GroupsNextLink argument to see them.):'
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.py:229]: Fixed

File: Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.py [hunk 2]
```diff
@@ -410,7 +412,7 @@ def main():
     tenant = params.get('tenant_id')
     auth_and_token_url = params.get('auth_id')
     enc_key = params.get('enc_key')
-    verify = not params.get('insecure', False)
+    verify = not params.get('insecure', True)
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.py:415]: By a mistake, fixed

File: Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups_test.py [hunk 3]
```diff
@@ -44,3 +46,23 @@ def test_commands(command, args, response, expected_result, mocker):
...
+    (list_members_command, {'group_id': 'under100'}, RESPONSE_LIST_MEMBERS_UNDER_100, EXPECTED_LIST_MEMBERS),
+    (list_members_command, {'group_id': 'above100'}, RESPONSE_LIST_MEMBERS_ABOVE_100, EXPECTED_LIST_MEMBERS)
+])  # noqa: E124
+def test_list_members_command(command, args, response, expected_result, mocker):
+    client = MsGraphClient(base_url='https://graph.microsoft.com/v1.0', tenant_id='tenant-id',
+                           auth_id='auth_and_token_url', enc_key='enc_key', app_name='ms-graph-groups',
+                           verify='use_ssl', proxy='proxies', self_deployed='self_deployed')
+    mocker.patch.object(client.ms_client, 'http_request', return_value=response)
+    mocker.patch.object(demisto, 'dt', return_value=RESPONSE_GET_GROUP)
+    result = command(client, args)
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups_test.py:61]: Fixed

File: Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups_test.py [hunk 4]
```diff
@@ -44,3 +46,23 @@ def test_commands(command, args, response, expected_result, mocker):
...
     result = command(client, args)
     assert expected_result == result[1]  # entry context is found in the 2nd place in the result of the command
+
+
+@pytest.mark.parametrize('command, args, response, expected_result', [
+    (list_members_command, {'group_id': 'under100'}, RESPONSE_LIST_MEMBERS_UNDER_100, EXPECTED_LIST_MEMBERS),
+    (list_members_command, {'group_id': 'above100'}, RESPONSE_LIST_MEMBERS_ABOVE_100, EXPECTED_LIST_MEMBERS)
+])  # noqa: E124
+def test_list_members_command(command, args, response, expected_result, mocker):
+    client = MsGraphClient(base_url='https://graph.microsoft.com/v1.0', tenant_id='tenant-id',
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups_test.py:56]: Done

File: Packs/MicrosoftGraphGroups/ReleaseNotes/1_0_12.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Azure Active Directory Groups
+- Fixed an issue where the ***list_members_command*** command did not handle properly more than 100 members in a single group.
```
[the remark this answers is not in the document]
Comment (dansterenson) [reply Packs/MicrosoftGraphGroups/ReleaseNotes/1_0_12.md:4]: Done

File: Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.py [hunk 6]
```diff
@@ -341,16 +342,19 @@ def list_members_command(client: MsGraphClient, args: Dict) -> Tuple[str, Dict,
...
+    if not group_data:
+        return_error('Could not find group data in the context, please run "!msgraph-groups-get-group" to retrieve group data.')
     if isinstance(group_data, list):
         group_data = group_data[0]
 
     if '@odata.nextLink' in members:
         next_link_response = members['@odata.nextLink']
         group_data['Members'] = members_outputs  # add a field with the members to the group
-        group_data['Members']['NextLink'] = next_link_response
+        group_data['MembersNextLink'] = next_link_response
```
Comment (reutshal) [inline Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.py:353]: Please update the command's outputs for both context paths you changes/added

Comment (dansterenson) [reply Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.py:353]: Done

**Review by reutshal — APPROVED** [review]

**Comment by reutshal** [comment]
@amshamah419 Can you please take a quick look? thanks

File: Packs/MicrosoftGraphGroups/ReleaseNotes/1_0_12.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Azure Active Directory Groups
+Fixed an issue where the ***msgraph-groups-list-members*** command did not handle properly more than 100 members in a single group.
```
Comment (amshamah419) [inline Packs/MicrosoftGraphGroups/ReleaseNotes/1_0_12.md:4]:
```suggestion
Fixed an issue where the ***msgraph-groups-list-members*** command did not handle more than 100 members in a single group properly.
```

**Pull request merged**